homeassistant/components/climate/radiotherm.py

```diff
@@ -77,19 +111,25 @@ class RadioThermostat(ClimateDevice):
     def __init__(self, device, hold_temp, away_temps):
         """Initialize the thermostat."""
         self.device = device
+        # It would be better if this was in update() since it triggers
+        # a network call but the thermostat will clear any temporary
+        # mode or temperature if this is called so we have to leave it here.
         self.set_time()
         self._target_temperature = None
         self._current_temperature = None
         self._current_operation = STATE_IDLE
         self._name = None
         self._fmode = None
+        self._fstate = None
```

Comment:
This is only assigned not used.

Reply:
I added because I want to support a binary sensor in the future to measure when the fan is actually on and off.  I'm not sure how to do that yet but having the fan state saved is part of it so I added it when updating all the other state variables.